modules/tinymce/src/plugins/quickbars/test/ts/browser/ContentEditableTest.ts

```diff
@@ -59,4 +60,27 @@ describe('browser.tinymce.plugins.quickbars.ContentEditableTest', () => {
     TinyAssertions.assertContentPresence(editor, { 'p[data-mce-bogus="all"]': 1, 'p[data-mce-caret="before"]': 1 });
     await pAssertToolbarNotVisible();
   });
+
+  it('TINY-9305: Toolbar is not shown when dragging', async () => {
+    const editor = hook.editor();
+    const emptyP = '<p>&nbsp;</p>';
+    editor.setContent(emptyP + emptyP + emptyP + '<p id="cefElement" contenteditable="false">CEF element</p>' + emptyP + emptyP + emptyP);
+    const elem = UiFinder.findIn(TinyDom.body(editor), '#cefElement').getOrDie();
+    Mouse.mouseDown(elem);
+    Mouse.mouseMoveTo(elem, 10, -75);
+    await Waiter.pWait(100);
```

Comment:
Why is the wait needed here add a comment or a waiter if possible.

Reply:
It was to be sure that the toolbar had time to render, but I checked and it is no needed (there is also a wait with 50ms in `pAssertToolbarNotVisible`), so I removed it